languages/python/exercises/concept/dicts/.docs/after.md

```diff
@@ -0,0 +1,5 @@
+Python dictionaries are very powerful and you will be using them a lot if you're working with APIs and such. Dictionaries are in their core just lists, but every value in the list has a name. This makes them super handy to use when you are working with data that is required on a request base and aren't expected to be in a specific order.
+
+Just like `lists`, dictionaries are unordered, mutable (which means you can change their content without changing their identity). They can also (just like `lists`) be nested, which means that a dictionary can contain another dictionary. The difference between lists and dictionaries and lists is that you access the data of a list depending on the location of the item in that list, with dictionaries you access the values using `keys` which are basically names for those values.
+
+You can get values from a dictionary by using the `.get(key, [default])`, which returns the value of the given key in the dictionary, if the key does not exist it returns the `default` value. Dictionaries also have the `.setdefault(key, [default])` method, which is almost the same as `.get()`, but it also places that key with the default value if it does not exist inside the dictionary.
```

Comment:
I think bringing up _collections_ is a good idea. Maybe just briefly, because otherwise, it might confuse the student. I'm not really familiar with _collections_ however. So just a few sentences about the subject and add it at the end?

Reply:
Yeah - something along the lines of the [collections module](https://docs.python.org/3/library/collections.html) offers the very useful [Counter](https://docs.python.org/3/library/collections.html#collections.Counter), [OrderedDict()](https://docs.python.org/3/library/collections.html#collections.OrderedDict), and [defaultdict()](https://docs.python.org/3/library/collections.html#collections.defaultdict), sub-types as extensions to the built-in dictionary.